src/helpers/transactionHelpers.ts

```diff
@@ -0,0 +1,26 @@
+import { UserTransactionTypes } from '../components/wallet/UserTransactionsTable'
+export const txParty = (address: string, tx: UserTransactionTypes) => {
+  const from = tx.fromAddr
+  const to = tx.toAddr
+  switch (tx.txType) {
+    case 'TRANSFER':
+      if (from === address) {
+        return { msg: 'send', label: 'to', address: to, color: 'error', op: '-' }
```

Comment:
Define a type for this object you return here and in other case statements below.